InvenTree/templates/account/base.html

```diff
@@ -49,10 +49,8 @@
 {% endblock %}
 </head>
 
-<body class='login-screen'>
-    <!--
-        Background Image Attribution: https://unsplash.com/photos/Ixvv3YZkd7w
```

Comment:
Suggestion: Maybe we should add this attribution to the credits in the docs?

Reply:
Yeah docs PR incoming